index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+<!-- The Links --> 
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="font-awesome/css/font-awesome.css">
+<link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;600&family=Poppins&family=Roboto&display=swap"
+      rel="stylesheet"
+    /> 
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+</head>
+<body>
+<!-- The Header -->   
+  <header class="header">  
+   <a href="#" class="logo">Gulalai Company </a>  
+    <button type="button"> <i class="fa fa-bars"></i>
+    </button> 
+  </header>
+  <section class="explain container">
+    <h1>Hey There.
```

Comment:
please fallow the heading order (h1,h2,h3,...) .its not bad but its good to screen readers to correctly read them.